--- fluidsynth-2.2.9.org/fluidsynth.conf.in     2022-06-29 11:32:41.484751424 +0200
+++ fluidsynth-2.2.9/fluidsynth.conf.in 2022-06-29 11:33:39.154377616 +0200
@@ -1,5 +1,5 @@
 # Mandatory parameters (uncomment and edit)
-#SOUND_FONT=@DEFAULT_SOUNDFONT@
+SOUND_FONT=/usr/share/soundfonts/default.sf2
 
 # Additional optional parameters (may be useful, see 'man fluidsynth' for further info)
 #OTHER_OPTS='-a alsa -m alsa_seq -p FluidSynth\ GM -r 48000'
