diff -up ./res/bnd/build-defaults.bnd.orig ./res/bnd/build-defaults.bnd
--- ./res/bnd/build-defaults.bnd.orig	2020-07-13 13:47:01.229077747 -0400
+++ ./res/bnd/build-defaults.bnd	2020-07-13 13:47:12.923095618 -0400
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.

-Bundle-Version: ${version_cleanup;${version}}
+Bundle-Version: ${version}

 Specification-Title: Apache Tomcat
 Specification-Version: ${version.major.minor}
